vhdbuilder/packer/components.json

```diff
@@ -255,29 +256,37 @@
       "multiArchVersions": [
         "v1.26.9",
         "v1.28.6",
+        "v1.28.6-1",
         "v1.29.3",
+        "v1.29.3-1",
         "v1.29.4"
       ]
     },
     {
       "downloadURL": "mcr.microsoft.com/oss/kubernetes-csi/azurefile-csi:*",
       "amd64OnlyVersions": [],
       "multiArchVersions": [
-        "v1.26.11",
+        "v1.26.11-2",
+        "v1.26.11-3",
         "v1.28.9",
+        "v1.28.9-1",
         "v1.29.3",
+        "v1.29.3-1",
         "v1.29.4",
-        "v1.30.0"
+        "v1.30.0",
+        "v1.30.0-1"
       ]
     },
     {
       "downloadURL": "mcr.microsoft.com/oss/kubernetes-csi/blob-csi:*",
       "amd64OnlyVersions": [],
       "multiArchVersions": [
-        "v1.21.7",
-        "v1.22.5",
+        "v1.21.7-2",
+        "v1.21.7-3",
```

Comment:
let's stop patch v1.21.* version since it's only for aks 1.26, and aks 1.26 will be not supported very soon

Reply:
updated, thanks!